README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+This charm supports deployment and scaling of a Project Clearwater system.  See http://www.projectclearwater.org for more information on Project Clearwater.
```

Comment:
There's a standard description of Project Clearwater used in READMEs - can you use that instead.